tests/scripts/list-identifiers.sh

```diff
@@ -1,4 +1,9 @@
-#!/bin/sh
+#!/bin/bash
+#
+# Outputs a file containing identifiers from internal header files or all
```

Comment:
This is a bit imprecise. The `stdout` result of this command is actually the one of `wc -l identifiers` at the bottom of the files, so we could also note it here. Additionally, maybe this could also state the name of the output file.

Reply:
Good point, I've made the description a bit more detailed.